doc/RELEASE-0.8.txt

```diff
@@ -0,0 +1,20 @@
+============
+Release v0.8
+============
+
+We are pleased to announce the release of screed v0.8. screed is a database
```

Comment:
Alright. It wasn't capitalized in the past release notes for screed.

On Sat, Mar 14, 2015, 00:31 Michael R. Crusoe <notifications@github.com>
wrote:

In doc/RELEASE-0.8.txt
<https://github.com/ged-lab/screed/pull/28#discussion_r26435272>:

> @@ -0,0 +1,20 @@ > +============ > +Release v0.8 > +============ > + >
+We are pleased to announce the release of screed v0.8. screed is a
database

Screed should be capitalized, unlike 'khmer'.


—
Reply to this email directly or view it on GitHub
<https://github.com/ged-lab/screed/pull/28/files#r26435272>.<img src="
https://ci6.googleusercontent.com/proxy/0_A0S38LYYksCKKyzCt8gfvDgr_UNd59jP56I-pkwXs-VQEpMRF0xHETtSRRCIGjuZIBL2Y5yNU2jtHzUZoU12i73l8cG41Fuc1Hsdu67Oo1DYAc4GXLKvdLHcH6PlncvYH5htDzIq8F_XowDpXLGbFKMIjY1w=s0-d-e1-ft#https://github.com/notifications/beacon/ACCvgjXBqglAfHg8vE85hu7mqA_3MRpjks5n07EngaJpZM4DuX8q.gif
">